Assets/SharedGameState.cs

```diff
@@ -20,15 +22,18 @@ public class SharedGameState : NetworkBehaviour
     public delegate void InstructorReceivedGameState();
     public static event InstructorReceivedGameState OnInstructorReceivedGameState = delegate { };
 
+   
     public override void OnNetworkSpawn()
     {
+       

```

Comment:
Just to make sure it sticks, please don't add superfluous line breaks to any file